aiida/engine/runners.py

```diff
@@ -67,7 +67,7 @@ def __init__(self, poll_interval=0, loop=None, communicator=None, rmq_submit=Fal
         self._plugin_version_provider = PluginVersionProvider()
 
         if communicator is not None:
-            self._communicator = communicator
+            self._communicator = plumpy.wrap_communicator(communicator, self._loop)
```

Comment:
hi @sphuber , as a reminder, since now you wrap the communicator here to `LoopCommnicator`, then there is no need to add task subscriber with scheduling task_receiver callback to the process event loop by `create_task`. 
In file `aiida/manage/manager.py::Manager::create_daemon_runner`  .

Reply:
Thanks a lot for the heads-up @unkcpz . I will have a look and update the PR.

Reply:
@unkcpz  I added this simplification in the first commit of this PR

Reply:
all good. Thanks!